tree/ntuple/v7/test/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ if(MSVC)
 endif()
 
 ROOT_ADD_GTEST(ntuple_basics ntuple_basics.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
+ROOT_ADD_GTEST(ntuple_bulk ntuple_bulk.cxx LIBRARIES ROOTDataFrame ROOTNTuple)
```

Comment:
```suggestion
ROOT_ADD_GTEST(ntuple_bulk ntuple_bulk.cxx LIBRARIES ROOTNTuple)
```

Reply:
In principle, yes, but the utility header `ntuple_test.hxx` depends on it. Perhaps this header can be split in another PR.